tool/update_llpc_test_checks.py

```diff
@@ -185,6 +184,11 @@ def main():
     else:
       common.SCRUB_TRAILING_WHITESPACE_TEST_RE = common.SCRUB_TRAILING_WHITESPACE_RE
 
+    tool_basename = ti.args.tool
+    tool_binary = tool_basename
+    if tool_basename == initial_args.tool and initial_args.tool_binary:
+      tool_binary = initial_args.tool_binary
```

Comment:
Isn’t `tool_binary` unconditionally overwritten in line 233?

Reply:
You're right, I wanted to move that but somehow didn't remove the original.